amoro-ams/amoro-ams-dashboard/pom.xml

```diff
@@ -113,4 +113,22 @@
             </plugin>
         </plugins>
     </build>
+    <profiles>
+        <profile>
+            <id>skip-webui-build</id>
```

Comment:
Maybe `skip-ams-dashboard-build` is a proper name as the module name is `amoro-ams-dashboard`.

Reply:
hi @zhoujinsong, `skip-ams-dashboard-build` seems too long for users. 

https://github.com/apache/flink/blob/b72ce532fdd77bca86a030be1b9878be4c097546/flink-runtime-web/pom.xml#L113

Reply:
Then how about `skip-dashbord-build`?

Reply:
> Then how about `skip-dashbord-build`?

Make sense, updated to `skip-dashboard-build`.